samples/cpp/ufacedetect.cpp

```diff
@@ -102,9 +84,9 @@ int main( int argc, const char** argv )
 
     cout << "old cascade: " << (cascade.isOldFormatCascade() ? "TRUE" : "FALSE") << endl;
 
-    if( inputName.empty() || (isdigit(inputName.c_str()[0]) && inputName.c_str()[1] == '\0') )
+    if( inputName.empty() || (isdigit(inputName[0]) && inputName[1] == '\0') )
```

Comment:
Same problem with `== '\0'` as in previous samples: should be replaced to `.size() == 1`.